src/test/kotlin/com/elpassion/mainframerplugin/integration/ResourceCopierTest.kt

```diff
@@ -0,0 +1,18 @@
+package com.elpassion.mainframerplugin.integration
+
+import com.elpassion.mainframerplugin.action.configure.templater.resourceCopier
+import com.intellij.testFramework.IdeaTestCase
+import java.io.File
+
+class ResourceCopierTest : IdeaTestCase() {
+
+    fun testShouldCopyFileFromSourceToDestination() {
+        val file = File("")
+        val copyDestination = "${file.absolutePath}${File.separator}justAFile"
```

Comment:
Is it worth using `File.separator` instead of `/` (working on Mac/Linux/Windows)?